corehq/apps/userreports/adapter.py

```diff
@@ -21,6 +21,9 @@ def drop_table(self):
     def refresh_table(self):
         raise NotImplementedError
 
+    def clear_table(self):
```

Comment:
For use in the future when we use the same data source, but want to test different data